NuKeeper.Inspection/NuGetApi/ApiPackageLookup.cs

```diff
@@ -17,9 +17,21 @@ public ApiPackageLookup(IPackageVersionsLookup packageVersionsLookup)
         public async Task<PackageLookupResult> FindVersionUpdate(
             PackageIdentity package,
             NuGetSources sources,
-            VersionChange allowedChange)
+            VersionChange allowedChange,
+            UsePrerelease usePrerelease)
         {
-            var allowBetas = package.Version.IsPrerelease;
+            bool allowBetas;
+
+            switch (usePrerelease)
```

Comment:
Extract a method here, and then you can do 
` case UsePrerelease.Always:  return true;` etc